run_client.sh.save

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
You probably didn't want to include this file? Looks like a backup file created by a text editor?